components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -15530,7 +15530,7 @@ public ByteArrayInputStream getGatewayPublishedAPIArtifacts(String APIId, String
     /**
      * Retrieve the list of blobs of the APIs for a given label
      *
-     * @param gatewayLabel - Gateway label of the API
+     * @param label - Gateway label of the API
```

Comment:
Adding a - (dash) is not correct. This will duplicate dash when the java doc is auto rendered ad displayed in the IDE when hovered on method call.

Reply:
Removed in the latest version